hail/python/hailtop/utils/utils.py

```diff
@@ -283,16 +283,21 @@ def is_transient_error(e):
     return [x async for x in agen]
 
 
-async def retry_forever(f, msg=None):
-    delay = 0.1
-    while True:
-        try:
-            await f()
-            break
-        except Exception as exc:
-            if msg:
-                log.info(msg(exc), exc_info=True)
-        await sleep_and_backoff(delay)
+def retry_forever(msg=None, error_logging_interval=10):
```

Comment:
I think I picked a bad name for this function. Can we change it with this PR to `retry_all_errors` because both this and `retry_transient_errors` retry forever, the real difference is which errors are retried.